server/knexfile.js

```diff
@@ -1,23 +0,0 @@
-// If we are not on a production server load the .env file

```

Comment:
NOTE: we still need the knex file to operate the cli. E.g when doing "knex migrate:latest|